src/rules/last-name-rules.ts

```diff
@@ -1,6 +1,5 @@
 export const LastNameRules: Array<Function> = [
-  v => !!v || 'A last name is required',
-  v => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
-  v => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+  v => !!v || 'Last name is required',
+  v => /^(?=.*\d)|(?=.*[a-z])|(?=.*[A-Z])/.test(v) || 'A firm name is required', // Contains at least a single character
```

Comment:
Aren't the name rules used in other components (where it may not be a firm name)?

Reply:
gah, copy pasta mistake, will fix for all these. good catch thanks!

Reply:
Updated in next commit